qcengine/programs/qchem.py

```diff
@@ -259,3 +263,201 @@ def parse_output(self, outfiles: Dict[str, str], input_model: "AtomicInput") ->
         output_data["success"] = True
 
         return AtomicResult(**{**input_model.dict(), **output_data})
+
+    @staticmethod
+    def parse_logfile(outtext: str) -> AtomicResult:
+        """Parses a log file. Should only be used when QCSCRATCH is not available."""
+        warnings.warn("parse_logfile should only be used when the QCSCRATCH directory is not available.")
+
+        NUMBER = "((?:[-+]?\\d*\\.\\d+(?:[DdEe][-+]?\\d+)?)|(?:[-+]?\\d+\\.\\d*(?:[DdEe][-+]?\\d+)?))"
```

Comment:
We need a common RE expression spot at some point.

Reply:
Okay, I made one.